recipes-openxt/vusb/files/xenclient-vusb.initscript

```diff
@@ -17,43 +17,40 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 #
 
-#
-# Starts vusb daemon.
-#
-
-PROG=/usr/sbin/vusb-daemon
+DESC="VUSBd"
+EXEC="/usr/bin/vusb-daemon"
+PIDFILE="/var/run/vusb.pid"
+OPTS=""
 
-# Make sure the progam exists
-[ -f "$PROG" ] || exit 0
-
-start() {
-	"$PROG" 2>&1 | logger -t 'vusb-daemon' &
```

Comment:
vusb-daemon prints to stderr and not syslog, so you need this pipe to logger.  That may make it incompatible with start-stop-daemon.

Reply:
Right, lets move that to another PR then.